developer-hub/src/reference_docs/safe_defensive_programming.rs

```diff
@@ -1 +1,376 @@
+//! As our runtime should _never_ panic; this includes eliminating the possibility of integer overflows,
+//! converting between number types, or even handling floating point usage with fixed point arithmetic
+//! to mitigate issues that come with floating point calculations.
 //!
+//! ## Integer Overflow
```

Comment:
The mod is called `Safe Defensive Programming` but only focuses on arithmetic. Maybe you can rename it to reflect that?

Reply:
I had planned to add more around general defensive practices within Substrate, but if it turns out to be majority-arithmetic I think we should rename it 👍